integration/monai/monai_nvflare/license.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
Do we need to include this file?
It it specified in header the Apache license can be found in https://www.apache.org/licenses/LICENSE-2.0

@chesterxgchen